src/Containers/Microsoft.NET.Build.Containers/Registry/Registry.cs

```diff
@@ -527,6 +525,17 @@ private async Task UploadBlobAsync(string repository, string digest, Stream cont
 
     }
 
+    public async Task PushManifestListAsync(string repositoryName, string[] tags, string manifestListJson, string mediaType, CancellationToken cancellationToken)
+    {
+        cancellationToken.ThrowIfCancellationRequested();
+        foreach (var tag in tags)
+        {
+            _logger.LogInformation(Strings.Registry_TagUploadStarted, tag, RegistryName);
+            await _registryAPI.Manifest.PutAsync(repositoryName, tag, manifestListJson, mediaType, cancellationToken).ConfigureAwait(false);
+            _logger.LogInformation(Strings.Registry_TagUploaded, tag, RegistryName);
+        }          
+    }
+
     public Task PushAsync(BuiltImage builtImage, SourceImageReference source, DestinationImageReference destination, CancellationToken cancellationToken)
```

Comment:
What is the difference between Push and Put in this context please?

Reply:
it was an existing naming convention here. "Push" image - a registry method, but "Put" manifest - a manifest method (an actual put request)